### DISCUSSION TITLE
spotify theming

### DISCUSSION THREAD

**Discussion by vyrx-dev** [prasanthrangan/hyprdots#1068] [Q&A] [answered]
how do add theming in Spotify like it is shown in the pictures?

**Comment by kRHYME7** [comment, marked as answer]
This is asked all the time, please use the search feature of github first.


Ow sometimes spicetify (the one applying the theme ) is annoying so you need some manual intervention 

I usually run this command to force it

```
spicetify backup apply || spicetify restore backup 
spicetify apply
```
(This is not used automatically as it is too slow to switch colors)
